.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: Release Cloudformation
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: Conventional Changelog Action
+        id: changelog
```

Comment:
There's no enforcement that pull requests are titled with conventional commit style, so you can't rely on this to be able to compute the next version correctly. 

The requirement of a single commit is a big headache for contributors so ideally you'd:

* Add another workflow that validates that the PR _title_ is conventional (or maybe just a job to the `Lint` workflow)
* Modify the repo settings to only allow "squash and merge" with the PR title as default commit message